manifests/params.pp

```diff
@@ -1,7 +1,9 @@
 # Reasonable defaults for all classes
 class r10k::params {
-  $postrun                = undef
-  $package_name           = ''
+  $package_name = $facts['os']['name'] ? {
+    'OpenBSD' => 'ruby31-r10k',
```

Comment:
This is not bad, but it's a separate enhancement to mention.

Reply:
I updated the PR title.Do you think that's enough or should I pull it into a seperate PR?